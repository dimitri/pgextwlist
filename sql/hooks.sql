set client_min_messages = debug;
set extwlist.custom_path = '/dummy';
set role mere_mortal;

-- in the hope refint stays at version 1.0 in PostgreSQL
create extension refint;
alter extension refint update;
comment on extension refint is 'snarky remark';
drop extension refint, refint;
