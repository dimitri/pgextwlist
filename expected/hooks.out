set client_min_messages = debug;
set extwlist.custom_path = '/dummy';
set role mere_mortal;
-- in the hope refint stays at version 1.0 in PostgreSQL
create extension refint;
DEBUG:  Considering custom script "/dummy/refint/before--1.0.sql"
DEBUG:  Considering custom script "/dummy/refint/before-create.sql"
DEBUG:  Considering custom script "/dummy/refint/after--1.0.sql"
DEBUG:  Considering custom script "/dummy/refint/after-create.sql"
alter extension refint update;
DEBUG:  Considering custom script "/dummy/refint/before--1.0--1.0.sql"
DEBUG:  Considering custom script "/dummy/refint/before-update.sql"
NOTICE:  version "1.0" of extension "refint" is already installed
DEBUG:  Considering custom script "/dummy/refint/after--1.0--1.0.sql"
DEBUG:  Considering custom script "/dummy/refint/after-update.sql"
comment on extension refint is 'snarky remark';
DEBUG:  Considering custom script "/dummy/refint/before-comment.sql"
DEBUG:  Considering custom script "/dummy/refint/after-comment.sql"
drop extension refint, refint;
DEBUG:  Considering custom script "/dummy/refint/before-drop.sql"
DEBUG:  Considering custom script "/dummy/refint/before-drop.sql"
DEBUG:  drop auto-cascades to function check_primary_key()
DEBUG:  drop auto-cascades to function check_foreign_key()
DEBUG:  Considering custom script "/dummy/refint/after-drop.sql"
DEBUG:  Considering custom script "/dummy/refint/after-drop.sql"
